src/index.html

```diff
@@ -5,8 +5,297 @@
+
+        <div
+          class="card" data-qa="card">
+          <img
+            class="card__image"
+            src="images/imac.jpeg"
+            alt="imac"
+          >
+          <h1 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+          <h3 class="card__code">Product code: 195434</h3>
+          <div class="card__raiting">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="card__reviews">Reviews: 5</p>
+          </div>
+          <div class="card__price">
+            <p class="card__price-price">Price:</p>
+            <p class="card__price-cost">$2,199</p>
+          </div>
+          <a
+            href="#"
+            class="card__buttom">
+          Buy
+          </a>
+        </div>
+
+        <div class="card" data-qa="card">
+          <img
+            class="card__image"
+            src="images/imac.jpeg"
+            alt="imac"
+          >
+          <h1 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+          <h3 class="card__code">Product code: 195434</h3>
+          <div class="card__raiting">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="card__reviews">Reviews: 5</p>
+          </div>
+          <div class="card__price">
+            <p class="card__price-price">Price:</p>
+            <p class="card__price-cost">$2,199</p>
+          </div>
+          <a
+            href="#"
+            class="card__buttom">
+          Buy
+          </a>
+        </div>
+
+        <div class="card" data-qa="card">
+          <img
+            class="card__image"
+            src="images/imac.jpeg"
+            alt="imac"
+          >
+          <h1 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+          <h3 class="card__code">Product code: 195434</h3>
+          <div class="card__raiting">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="card__reviews">Reviews: 5</p>
+          </div>
+          <div class="card__price">
+            <p class="card__price-price">Price:</p>
+            <p class="card__price-cost">$2,199</p>
+          </div>
+          <a
+          href="#"
+          class="card__buttom">
+        Buy
```

Comment:
Remember about code style. Check it everywhere
```suggestion
          <a
            href="#"
            class="card__buttom"
          >
            Buy
```